### PR TITLE
dev_parrot

### PR DESCRIPTION
**Pull request by Jegama**

### CODE REVIEW

**Comment by jmhobbs** [comment]
Dig it!  Could we make it more general as "Laptop Parrot" or something?

**Comment by Jegama** [comment]
Just changed it :)

File: parrots.json [hunk 1]
```diff
@@ -561,5 +561,9 @@
     {
       "hd": "hd/serbiaparrot.gif",
       "name": "Serbian Parrot"
-    }
+    },
+    {
+      "hd": "hd/laptop_parrot.gif",
+      "name": "Laptop Parrot"
+    },
```
Comment (jmhobbs) [inline parrots.json]: Oops! Extra comma needs to be trimmed off!

Comment (Jegama) [reply parrots.json]: My bad, just fixed it :)

**Comment by jmhobbs** [comment]
Thanks a bunch!

**Pull request merged**